core/src/main/java/com/graphhopper/routing/util/WheelchairFlagEncoder.java

```diff
@@ -75,12 +75,15 @@ protected WheelchairFlagEncoder(int speedBits, double speedFactor) {
         safeHighwayTags.add("service");
         safeHighwayTags.add("platform");
 
-        excludeHighwayTags.add("trunk");
-        excludeHighwayTags.add("trunk_link");
-        excludeHighwayTags.add("primary");
-        excludeHighwayTags.add("primary_link");
-        excludeHighwayTags.add("steps");
-        excludeHighwayTags.add("track");
```

Comment:
`excludeHighwayTags` wasn't properly working anyway, so I removed this difference and it is now using the same highways foot uses.